modules/oadp-release-notes-1-3-1.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes-1-3.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="oadp-release-notes-1-3-1_{context}"]
+= OADP 1.3.1 release notes
+
+The {oadp-first} 1.3.1 release notes lists new features and resolved issues.
+
+[id="new-features-1-3-1_{context}"]
+== New features
+
+.OADP 1.3 Data Mover is now fully supported
+
+The OADP built-in Data Mover, introduced in OADP 1.3 as a Technology Preview, is now fully supported for both containerized and virtual machine workloads.
+
+.{ibm-cloud-name} S3 backup storage provider is now supported
+
+Previously, {ibm-cloud-name} was one of the AWS S3 compatible object storage provider, which was unsupported for backup storage locations.
+With this update, {ibm-cloud-name} S3 is now supported as an AWS S3 compatible backup storage provider.
+
+[id="resolved-issues-1-3-1_{context}"]
+== Resolved issues
+
+.OADP operator now correctly reports the missing region error
+
+Previously, when a user specified `profile:default` without specifying the `region` in AWS Backup Storage Location (BSL) configuration, the OADP operator failed to report the `missing region` error on the Data Protection Application (DPA) custom resource (CR). This update corrects validation of DPA BSL specification for the AWS provider. As a result, the OADP operator reports the `missing region` error.
+
+link:https://issues.redhat.com/browse/OADP-3044[OADP-3044]
+
+.Custom labels are not removed from the namespace, where OADP is installed
+
+Previously, the `openshift-adp-controller-manager` pod would reset the labels on the namespace, where OADP is installed. Consequently, this caused synchronization issues for applications requiring custom labels such as Argo CD, leading to improper functionality. With this update, this issue is fixed and custom labels are not removed from the namespace, where OADP is installed.
+
+link:https://issues.redhat.com/browse/OADP-3189[OADP-3189]
+
+.OADP `must-gather` image collects CRD's
+
+Previously, the OADP `must-gather` image did not collect the Custom Resource Definitions (CRD's) shipped by OADP. Consequently, it was not possible to use the `omg` tool to extract data on the support shell. For example, the `omg get backup` command did not work because `must-gather` was unable to collect CRD's.
+With this fix, `must-gather` now collects CRD's shipped by OADP and can use the `omg` tool to extract data.
+
+link:https://issues.redhat.com/browse/OADP-3229[OADP-3229]
+
+.Garbage collection has the required default frequency
+
+Previously, the `garbage-collection-frequency` field had a wrong description for the default frequency. With this update, the `gc-controller` correctly executes its reconciliation loop every hour by default. As a result, garbage collection now has the expected default frequency.
```

Comment:
the issue was with the description only
not the actaul value

Reply:
thank you, @amastbau, for reviewing it. I have updated it in this [commit](https://github.com/openshift/openshift-docs/pull/74026/commits/d39f89b0d244383f5e71b781477e0e98c44910a7 ).